lib/i18n/tasks/translators/deepl_translator.rb

```diff
@@ -41,11 +41,11 @@ def options_for_translate_values(**options)
     end
 
     def options_for_html
-      { tag_handling: 'xml' }
+      { tag_handling: 'xml', ignore_tags: %w[i18n] }
     end
 
     def options_for_plain
-      { preserve_formatting: true }
+      { preserve_formatting: true, tag_handling: 'xml', ignore_tags: %w[i18n] }
```

Comment:
Do we now need to html-escape/unescape plain text?

Reply:
I added a suggestion on escaping + unescaping now 🙂